### PR TITLE
Do not prompt for private and public address on upgrade

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: install_scripts/templates/replicated-operator.sh [hunk 1]
```diff
@@ -433,22 +433,23 @@ if [ -z "$PUBLIC_ADDRESS" ] && [ "$AIRGAP" != "1" ] && [ "$NO_PUBLIC_ADDRESS" !=
...
-            PUBLIC_ADDRESS="$REPLICATED_OPTS_VALUE"
-            printf "The installer will use service address '%s' (imported from $CONFDIR/replicated-operator 'PUBLIC_ADDRESS')\n" $PUBLIC_ADDRESS
-        fi
+        PUBLIC_ADDRESS="$REPLICATED_OPTS_VALUE"
     fi
-
-    if [ -n "$PUBLIC_ADDRESS" ]; then
-        shouldUsePublicIp
+    if [ -n "$REPLICATED_OPTS_VALUE" ] && [ "$REPLICATED_OPTS_VALUE" = "$PUBLIC_ADDRESS" ]; then
+        PUBLIC_ADDRESS="$REPLICATED_OPTS_VALUE"
```
Comment (areed) [inline install_scripts/templates/replicated-operator.sh:441]: I don't understand how you could be in this block without them already being equal

Comment (emosbaugh) [reply install_scripts/templates/replicated-operator.sh:441]: if the public ip from discoverPublicIp line 434 does not match the one from /etc/default/replicated-operator

Comment (areed) [reply install_scripts/templates/replicated-operator.sh:442]: The 2nd test in line 440 checks they're equal so the assignment in 441 is redundant.

File: install_scripts/templates/replicated-2.0.sh [hunk 2]
```diff
@@ -725,22 +729,25 @@ if [ -z "$PUBLIC_ADDRESS" ] && [ "$AIRGAP" != "1" ] && [ "$NO_PUBLIC_ADDRESS" !=
...
-                printf "The installer will use service address '%s' (imported from $CONFDIR/replicated-operator 'PUBLIC_ADDRESS')\n" $PUBLIC_ADDRESS
-            fi
+            PUBLIC_ADDRESS="$REPLICATED_OPTS_VALUE"
         fi
-
-        if [ -n "$PUBLIC_ADDRESS" ]; then
-            shouldUsePublicIp
+        # Check that the public address from discoverPublicIp matches the one from Replicated Operator opts
+        if [ -n "$REPLICATED_OPTS_VALUE" ] && [ "$REPLICATED_OPTS_VALUE" = "$PUBLIC_ADDRESS" ]; then
+            PUBLIC_ADDRESS="$REPLICATED_OPTS_VALUE"
```
Comment (areed) [inline install_scripts/templates/replicated-2.0.sh:738]: this one too

**Pull request merged**